ops.go

```diff
@@ -333,6 +329,11 @@ func (o *ImageOps) Transform(d Decoder, opt *ImageOptions, dst []byte) ([]byte,
 
 		frameCount++
 
+		// break out if we're creating a single frame and we've already done one
+		if opt.DisableAnimatedOutput && frameCount > 0 {
```

Comment:
We should be able to remove the `frameCount > 0` check since `frameCount` has just been incremented:

```suggestion
		if opt.DisableAnimatedOutput {
```

Reply:
Good catch, tunnel vision, updated!